examples/lighting-app/nrfconnect/README.md

```diff
@@ -100,7 +101,7 @@ The example supports building and running on the following devices:
 
 This section lists the User Interface elements that you can use to control and
 monitor the state of the device. All these elements can be located on the
-following board picture:
+following development kit picture:
 
 ![nrf52840 DK](../../platform/nrfconnect/doc/images/nrf52840-dk.jpg)
```

Comment:
The sentence "All these elements..." and the picture need to be removed for all examples, or nRF5340 picture needs to be added.